GitUI/CommandsDialogs/FormMergeBranch.cs

```diff
@@ -110,7 +111,7 @@ private void OkClick(object sender, EventArgs e)
                                                                                                    addLogMessages.Checked ? (int)nbMessages.Value : (int?)null));
 
             var wasConflict = MergeConflictHandler.HandleMergeConflicts(UICommands, this, !noCommit.Checked);
-
+            ScriptManager.RunEventScripts(this, ScriptEvent.AfterMerge);
```

Comment:
Shouldn't this run only if it is successfully merged?

Reply:
You are right. This count be better.